shiksha-website/shiksha-frontend/src/app/layout/header/header.component.html

```
@@ -1,7 +1,8 @@
 <header
   class="fixed z-30 top-0 left-0 md:left-[270px] right-0 header shadow-md py-2 px-4 md:px-8 flex justify-between lg:justify-end items-center"
 >
-  <button class="p-2 rounded-md lg:hidden" (click)="openSidebar()">
+  <button class="p-2 rounded-md lg:hidden" (click)="openSidebar()"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)